static/third/jquery-filedrop/jquery.filedrop.js

```diff
@@ -362,7 +362,7 @@
 
         // Pass any errors to the error option
         if (xhr.status < 200 || xhr.status > 299) {
-          if (this.responseText.includes("Upload would exceed your maximum quota.")) {
+          if (this.responseText.includes("Upload would exceed your organization's maximum quota.")) {
```

Comment:
s/maximum/upload?

I'm a little confused how this works though, given translations.